cmd/kpod/load.go

```diff
@@ -104,6 +110,16 @@ func loadImage(store storage.Store, opts loadOptions) error {
 	systemContext := common.GetSystemContext("")
 
 	src := dockerArchive + opts.input
+	loadOpts := libkpodimage.CopyOptions{
+		Quiet:    opts.quiet,
+		OciImage: opts.image,
+	}
 
-	return libkpodimage.PullImage(store, src, false, opts.quiet, systemContext)
+	if err := libkpodimage.PullImage(store, src, false, loadOpts, systemContext); err != nil {
```

Comment:
Should any of these other options be copy options?  Or is this something specific?

Reply:
Cleaned it up a bit.